src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -1593,10 +1686,10 @@ BlankNode Visitor::visitTypesafe(Parser::BlankNodeContext* ctx) {
 
 template <typename Ctx>
 auto Visitor::visitVector(const std::vector<Ctx*>& childContexts)
-    -> std::vector<decltype(visitTypesafe(childContexts[0]))> {
-  std::vector<decltype(visitTypesafe(childContexts[0]))> children;
+    -> std::vector<decltype(visit(childContexts[0]))> {
+  std::vector<decltype(visit(childContexts[0]))> children;
   for (const auto& child : childContexts) {
-    children.emplace_back(std::move(visitTypesafe(child)));
+    children.emplace_back(std::move(visit(child)));
```

Comment:
That `std::move` is redundant here, because the `visit` functions all return values, so this already is a temporary.